akka-remote/src/main/resources/reference.conf

```diff
@@ -755,8 +755,18 @@ akka {
         #   "<getHostName>"      InetAddress.getLocalHost.getHostName
         #
         hostname = ""
+
+        # Number of times to check the Aeron media driver that the inbound channel
+        # is active
+        retries = 20
+
+        # Time to wait between each retry
+        retry-interval = 200 ms
```

Comment:
Maybe make this a single `bind-timeout` setting or something along those lines instead, and remove the configuration for retries/interval, as a user that would be what I may want to configure upwards if my system is slow to bind?

Reply:
I agree. Users don't care about how often we check the counters. They care about the total timeout. The retry sleep can be hardcoded.

Reply:
Agreed, changed

Reply:
:+1: